app/lib/frontend/templates/views/pkg/labeled_scores.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import '../../../dom/dom.dart' as d;
+
+d.Node renderLabeledScoresNode({
```

Comment:
Perhaps name this without the `render`-prefix, and reserve that for the actual render to a string:
 ```suggestion
d.Node labeledScoresNode({
```

Reply:
I like that!